src/core/templates/mail/account_approved.txt

```diff
@@ -0,0 +1,14 @@
+Dear {{ user_name }},
+
+Thank you for registering to use the Altmetrics service.
+
+This email is to let you know that your account has been approved.
+{% if tech_email %}
+
+    If you have any questions, please feel free to contact our support team at {{ tech_email }}.
```

Comment:
If this is a plain text template, will the line above end up indented in the rendered email?

Reply:
hmmm, apparently. Well spotted!